input/ignoreWarnings.txt

```diff
@@ -1,6 +1,28 @@
 == Suppressed Messages ==
 
-# Add warning and/or information messages here after you've confirmed that they aren't really a problem
-# instructions for ignoreWarnings.txt https://confluence.hl7.org/pages/viewpage.action?pageId=66938614#ImplementationGuideParameters-ManagingWarningsandHints
-# (And include comments like this justifying why)
```

Comment:
Don't remove the instructions here, this is a sample IG, part of it's purpose is to provide these types of instructions